src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -270,12 +270,13 @@ struct kds_cu_info
       boost::char_separator<char> sep(",");
       tokenizer tokens(line, sep);
 
-      if (std::distance(tokens.begin(), tokens.end()) != 5)
+      if (std::distance(tokens.begin(), tokens.end()) != 6)
```

Comment:
Please update the example information comments on line 266-268 for these is 1 more element being looked at.